runtime/src/main/java/org/corfudb/runtime/view/replication/QuorumReplicationProtocol.java

```diff
@@ -117,6 +117,15 @@ public ILogData peek(RuntimeLayout runtimeLayout, long address) {
                 .collect(Collectors.toMap(SimpleImmutableEntry::getKey, SimpleImmutableEntry::getValue));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void commitAll(RuntimeLayout runtimeLayout, Collection<Long> addresses) {
+        // TODO: parallelize this operation as in chain replication.
```

Comment:
get rid of todo

Reply:
This class is not being used, so not implementing for now, just made it compile.